docs/sdn_controller.md

```diff
@@ -37,6 +37,7 @@ If none is provided, the plugin will create its own self-signed certificates.
 ## Requirements
 
 > All requirements are met by running up to date XCP-ng hosts.
+
```

Comment:
Check [this](https://spec.commonmark.org/dingus/?text=%3E%20foo%0A%3E%0A%3E%20bar%0A) vs [that](https://spec.commonmark.org/dingus/?text=%3E%20foo%0A%0A%3E%20bar%0A).